<center>
    <img src="https://cf-courses-data.s3.us.cloud-object-storage.appdomain.cloud/IBMDeveloperSkillsNetwork-DS0105EN-SkillsNetwork/labs/Module2/images/SN_web_lightmode.png" width="300" alt="cognitiveclass.ai logo">
</center>


#### Add your code below following the instructions given in the course


# My Jupyter Notebook on IBM Watson Studio

**Francois Joubert**
Data Scientist

*I am interested in data science because of the clarity that data can give. The proof is in the pudding, or data in this case and I am fascinated by the interesting trends and insights that can be identified when taking a deeper look at the data.*

### The code should formulate the first 10 numbers in the Fibonacci sequence

In [1]:
fib <- numeric(10)
fib[1] <- 1
fib[2] <- 1
for (i in 3:10) {
  fib[i] <- fib[i-1] + fib[i-2]
}
fib

[1]  1  1  2  3  5  8 13 21 34 55

In [2]:
ratios <- fib[-1]/fib[-10]

ratios

[1] 1.000000 2.000000 1.500000 1.666667 1.600000 1.625000 1.615385 1.619048
[9] 1.617647

This sequence converges to a common factor:
1. 1
2. 2
3. 1.5
4. 1.66667
5. 1.6
6. 1.625
7. 1.6153846
8. 1.61904761904762
9. 1.61764705882353

______________________________________________________________________________________________________________

The ratio is calculated by dividing the newest element by the previous element:
```
ratios <- fib[-1]/fib[-10]```